pyanaconda/modules/storage/constants.py

```diff
@@ -25,6 +25,28 @@
     ISCSI_INTERFACE_IFACENAME
 
 
+INCONSISTENT_SECTOR_SIZES_SUGGESTIONS = (
```

Comment:
Yes, we translate the suggestions if we want to. Just a note, we are not able to translate the whole error message.

Reply:
I know that not everything could be translated because some parts come from outside. However, this part should be marked and also other messages like "Failed to add a device:".